components/VC/Views/VCCardView.tsx

```diff
@@ -68,27 +72,60 @@ export const VCCardView: React.FC<VCItemProps> = props => {
             ? Theme.Styles.selectedBindedVc
             : Theme.Styles.closeCardBgContainer
         }>
-        <VCCardViewContent
-          vcMetadata={props.vcMetadata}
-          context={context}
-          walletBindingResponse={walletBindingResponse}
-          credential={vc}
-          verifiableCredentialData={verifiableCredentialData}
-          fields={fields}
-          wellknown={wellknown}
-          generatedOn={formattedDate}
-          selectable={props.selectable}
-          selected={props.selected}
-          service={service}
-          isPinned={props.isPinned}
-          onPress={() => props.onPress(service)}
-          isDownloading={props.isDownloading}
-          flow={props.flow}
-          isKebabPopUp={isKebabPopUp}
-          DISMISS={DISMISS}
-          KEBAB_POPUP={KEBAB_POPUP}
-          isVerified={props.vcMetadata.isVerified}
-        />
+        {props.isTopCard ? (
+          <CopilotStep
+            text={t('copilot:cardMessage')}
+            order={6}
+            name={t('copilot:cardTitle')}>
+            <CopilotView>
+              <VCCardViewContent
```

Comment:
Can we extract this VCCardViewContent to an internal function or constant, this would make it easy to maintain consistency. Say We make some change in this VCCardViewContent we need not change in 2 places everytime

Reply:
created a internal function